PrismFormsGallery/Source/Modules/CommandingModule/CommandingModule.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+ <ItemGroup>
+    <PackageReference Include="Prism.Forms" Version="7.2.0.1347-pre" />
```

Comment:
Please update this to the stable Prism 7.2

Reply:
Done